scripts/test/maven/runTests.sh

```diff
@@ -58,7 +58,7 @@ runMavenQuickSystemTests() {
     echo "Tearing down instance "$SHORT_ID" and "$SHORT_ID-main""
     echo "-------------------------------------------------------------------------------"
     "$SCRIPTS_DIR/test/maven/tearDown.sh" "$SHORT_ID" "$SHORT_ID-main" > "$OUTPUT_DIR/$TEST_NAME.tearDown.log"
-    echo "Tearing down for instance "$SHORT_ID" and "$SHORT_ID-main complete"
+    echo "Tearing down for instance "$SHORT_ID" and "$SHORT_ID-main complete""
```

Comment:
There's a double quote missing after `"$SHORT_ID-main`, but as it is, I think it won't include any double quotes in the printed line. Bash will interpret it as closing the string literal, and then since there's no space I think it's joining the variable onto it outside the double quotes. You can escape the double quotes with a slash, like this:

```bash
echo "Tearing down for instance \"$SHORT_ID\" and \"$SHORT_ID-main\" complete"
```

This also applies to the other echo above:

```bash
    echo "Tearing down instance \"$SHORT_ID\" and \"$SHORT_ID-main\""
```

Reply:
Updated as requested